docs/UserGuide.md

```diff
@@ -32,33 +31,50 @@
+We have just the right solution for you!<br><br>
+Introducing **Nav@NUS**, you new navigation assistant!
+Nav@NUS is a useful command line interface (CLI) application to guide you in navigating around the NUS campus
 via the school's shuttle services. This application enables you to retrieve key bus information easily, skipping the 
-hassle of physically checking the bus stop's notice board. Nav@NUS brings convenience to you and wishes your commute in
-NUS to be as effortless as possible. Nav@NUS uses a CLI to facilitate quick typing and retrieval of
+hassle of physically checking the bus stop's notice board.Nav@NUS is a tool tailor made for anyone unfamiliar to 
+NUS  campus,students,professors and visitors included. Nav@NUS brings convenience to you and wishes your 
+commute in NUS to be as effortless as possible. Nav@NUS uses a CLI to facilitate quick typing and retrieval of 
 information that you require.
 
 Nav@NUS consists of 3 main features:
 
-* *Route*: Searches for bus routes from your location to your intended destination.
-* *Dine*: Seeks dining options for you to explore culinary world of NUS.
-* *Fav*: Saves your commands for you to have a personalised user experience catered to your needs.
+* **Route**: Searches for bus routes from your location to your intended destination.
+* **Dine**: Seeks dining options for you to explore culinary world of NUS.
+* **Fav**: Saves your commands for you to have a personalised user experience catered to your needs.
 
 Skip the tight squeeze near information boards and use Nav@NUS today!
 
 ## 2. Quick Start
-The following steps will guide you through the process of running Nav@NUS.
-
-1. Ensure that you have Java `11` or above installed in your computer.
-2. Download the latest `duke.jar` from [here](https://github.com/AY2021S1-CS2113T-F14-3/tp/releases).
-3. Copy the file to the folder you want to use as the _home folder_ for duke.jar bus application.
-4. In command prompt, `cd` to the directory of the _home folder_.
-5. Run the .jar file in the command prompt as follows: `java -jar duke.jar`.
-6. Type `help` to view commands available.
-7. Refer to [Features](#features) below for details of each command.
+The following steps will guide you through the process of running **Nav@NUS**.
+
+1. Ensure that you have Java `11` or above installed in your computer. If you do not have it installed,
+follow the guide [here](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-microsoft-windows-platforms.html#GUID-A7E27B90-A28D-4237-9383-A58B416071CA).
+2. Download the latest `Nav@NUS.jar` from [here](https://github.com/AY2021S1-CS2113T-F14-3/tp/releases).
+3. Open command prompt on your computer.
+Orientate yourself to the command line interface. As seen in the figure below, the red arrow points to
+where you have to type in commands.
+![Windows CLI](UG_Figures/windowsCLI.png)
+Figure 1.1 Windows CLI
```

Comment:
Maybe remove the figure numbers for now, since we removed it from other places also, if the figure is right below the content
```suggestion
```